connector_jira/tests/common.py

```diff
@@ -63,11 +63,11 @@
 import os
 from contextlib import contextmanager
 from os.path import dirname, join
+from unittest import mock
 
-import mock
 from vcr import VCR
 
-from odoo.addons.component.tests.common import SavepointComponentCase
```

Comment:
`SavepointComponentCase` removed in component module, use TransactionComponentCase instead